functions/Remove-DbaDbUser.ps1

```diff
@@ -0,0 +1,230 @@
+                            }
+                        }
+                    }
+
+                    # Perform the changes with the owner alters first so function can exit w/o a drop if -Force wasn't used
+                    foreach ($schema in $alterSchemas) {
+                        Write-Message -Level Verbose -Message "Owner of Schema $schema will be changed to [dbo]."
+                        if ($PSCmdlet.ShouldProcess($server, "Change the owner of Schema $schema to [dbo].")) {
+                            $schema.Owner = "dbo"
+                            $schema.Alter()
+                        }
+                    }
+
+                    foreach ($schema in $dropSchemas) {
+                        if ($PSCmdlet.ShouldProcess($server, "Drop Schema $schema from Database $db.")) {
+                            $schema.Drop()
+                        }
+                    }
+                }
+
+                # Drop the User
+                if ($PSCmdlet.ShouldProcess($server, "Drop User $user from Database $db.")) {
+                    try {
+                        $user.Drop()
+                        [pscustomobject]@{
+                            SqlInstance = $server.name
+                            Database    = $db.name
+                            User        = $user
+                            Status      = "Dropped"
+                        }
+                    } catch {
+                        Write-Message -Level Verbose -Message "Could not drop $user from Database $db on target $server"
+                        [pscustomobject]@{
+                            SqlInstance = $server.name
+                            Database    = $db.name
+                            User        = $user
+                            Status      = "Not Dropped"
+                        }
+                    }
+                }
+            }
+		}
+	}
+    
+    process {
+        if ($UserCollection) {
+            Remove-DbUser $UserCollection
+        } else {
+            foreach ($instance in $SqlInstance) {
+                try {
+                    Write-Message -Level Verbose -Message "Connecting to $instance"
+                    $server = Connect-SqlInstance -SqlInstance $instance -SqlCredential $sqlcredential
+                }
+                catch {
+                    Stop-Function -Message "Failure" -Category ConnectionError -ErrorRecord $_ -Target $instance -Continue
+                }
+                
+                $databases = $server.Databases
+                
+                if ($Database) {
+                    $databases = $databases | Where-Object Name -In $Database
+                }
+                if ($ExcludeDatabase) {
+                    $databases = $databases | Where-Object Name -NotIn $ExcludeDatabase
+                }
+                
+                foreach ($db in $databases) {
+                    Write-Message -Level Verbose -Message "Get users in Database $db on target $server"
```

Comment:
Need to just filter out databases that are not accessible on line 211 or validate if it is accessible here. I think our pattern is to filter them out when we build the `$databases` variable in most commands.